pinot-core/src/test/java/org/apache/pinot/queries/CastQueriesTest.java

```diff
@@ -130,6 +146,18 @@ public void testCastSum() {
     assertEquals(((Number) aggregationResult.get(1)).intValue(), NUM_RECORDS / 4);
   }
 
+  @Test
+  public void testCastMV() {
```

Comment:
Add an `ID_SET` related test here to ensure the functionality we want to support for the internal use case works.

Reply:
Done.